pkg/3scale/amp/component/apicast_monitoring.go

```diff
@@ -155,17 +155,41 @@ func ApicastPrometheusRules(ns string) *monitoringv1.PrometheusRule {
 					Name: fmt.Sprintf("%s/apicast.rules", ns),
 					Rules: []monitoringv1.Rule{
 						{
-							Alert: "ApicastDroppedConnections",
+							Alert: "ApicastRequestTime",
 							Annotations: map[string]string{
-								"summary":     "{{$labels.container_name}} replica controller on {{$labels.namespace}}: Has more than 10 dropped connections in the last 5 minutes",
-								"description": "{{$labels.container_name}} replica controller on {{$labels.namespace}} project: Has more than 10 dropped connections in the last 5 minutes",
+								"summary":     "Request on instance {{ $labels.instance }} is taking more than one second to process the requests",
+								"description": "High number of request taking more than a second to be processed",
 							},
-							Expr: intstr.FromString(fmt.Sprintf(`sum (increase(nginx_http_connections{namespace="%s",job=~"apicast-(production|staging)-monitoring",state="accepted"}[5m])) - sum (increase(nginx_http_connections{namespace="%s",job=~"apicast-(production|staging)-monitoring",state="handled"}[5m])) > 10`, ns, ns)),
+							Expr: intstr.FromString(fmt.Sprintf(`sum(rate(total_response_time_seconds_bucket{namespace='%s', pod=~'apicast-staging.*'}[1m])) - sum(rate(upstream_response_time_seconds_bucket{namespace='%s', pod=~'apicast-staging.*'}[1m])) > 1`, ns, ns)),
```

Comment:
maybe here, we should move from staging to staing|production no?

Reply:
I do not really have a strong opinion. 

Does it make sense to have alerts on staging gateway? I do not know. 

Up to you. Drop here the correct expresion or update this branch yourself.

Reply:
Yeah, maybe does not make sense at all. At the alert only on prod

Reply:
applied